tests/unit/services/apigateway/test_handler_integration_request.py

```diff
@@ -266,6 +263,33 @@ def test_multivalue_mapping(
         assert context.integration_request["headers"]["multi"] == "header1,header2"
         assert context.integration_request["query_string_parameters"]["multi"] == ["qs1", "qs2"]
 
+    def test_integration_uri_path_params_undefined(
+        self, integration_request_handler, default_invocation_request, create_context
+    ):
+        context = create_context(request=default_invocation_request)
+        context.resource_method["methodIntegration"]["requestParameters"] = {
+            "integration.request.path.path": "method.request.path.wrongvalue",
+        }
+        context.resource_method["methodIntegration"]["uri"] = "https://example.com/{path}"
+        integration_request_handler(context)
+        assert context.integration_request["uri"] == "https://example.com/{path}"
```

Comment:
Has this behavior been tested against AWS? Should we raise an error or let the integration fail?

Reply:
It has been tested yes! It is the integration itself which is failing, it says the URI contains invalid characters or something similar, I guess it's up to the HTTP client then to validate.